modules/virt-editing-vm-web.adoc

```diff
@@ -32,7 +32,7 @@ endif::[]
 
 Edit select values of a {object} in the web console by clicking the pencil icon next to the relevant field. Other values can be edited using the CLI.
 
-Labels and annotations are editable for both preconfigured Red Hat templates and your custom virtual machine templates. All other values are editable only for custom virtual machine templates that users have created using the Red Hat templates or the *Create Virtual Machine Template* wizard.
+Labels and annotations are editable for both preconfigured Red Hat templates and your custom virtual machine (VM) templates. All other values are editable only for custom virtual machine templates that users have created using the Red Hat templates or the *Create Virtual Machine Template* wizard.
```

Comment:
There is no 'Create Virtual Machine Template' wizard in 4.11 now, it can be removed from here.